### PR TITLE
PLF-8437 : Duplication sub-comments on lecko extract

### PR DESCRIPTION
**Pull request by slickexo3**
# FIX - Duplicates sub comments activity on lecko extracts

Fix duplicates lines during a Lecko extraction.
The tests corresponding to the extraction of comments have been updated.

### CODE REVIEW

File: service/src/test/java/org/exoplatform/lecko/service/TestUserActivityWithCommentAComment.java [hunk 1]
```diff
@@ -92,6 +85,9 @@ public void testBuildUserExportWithCommentAComment() throws Exception {
     String ls = System.getProperty("line.separator");
     String[] lines = fileContent.split(ls);
 
+    // Assert that the file contain 9 lines.
+    assertEquals(9, lines.length);
```
Comment (rdenarie) [inline service/src/test/java/org/exoplatform/lecko/service/TestUserActivityWithCommentAComment.java:89]: I think we should update the test TestSpaceActivityWithCommentAComment with the same assert.

**Review by rdenarie — CHANGES_REQUESTED** [review]

**Comment by fdrouet** [comment]
Hello @slickexo3, thank you for your first contribution to eXo Platform.

In order to accept your contribution, we need you to sign a Contributor License Agreement : http://developer.exoplatform.org/#id-community-contributions

**Comment by fdrouet** [comment]
> Hello @slickexo3, thank you for your first contribution to eXo Platform.
> 
> In order to accept your contribution, we need you to sign a Contributor License Agreement : http://developer.exoplatform.org/#id-community-contributions

CLA received

**Review by rdenarie — APPROVED** [review]

**Comment by boubaker** [comment]
@rdenarie please recreate it on https://github.com/exo-addons/lecko

**Pull request closed without merging**